src/utilities/project-utils.ts

```diff
@@ -10,109 +10,231 @@ import pcLogo from "../assets/project-images/pc-logo.png";
   url: string;
+  footer: string;
+  footerLogo: Array<any>;
 }
 
 export const ProjectTypes: ProjectType[] = [
   {
     id: 1,
     name: "Molecular Workbench",
-    logo: mwLogo,
+    projectLogo: mwLogo,
     url: "http://mw.concord.org/nextgen/",
+    footer: 
+    "<p class=\"footer-txt\">This open educational resource from the Concord Consortium is free for use under the Creative Commons Attribution International license (<a title=\"\" href=\"http://creativecommons.org/licenses/by/4.0/\" target=\"\">CC-BY 4.0</a>) and powered by open source software packages. For info about our licenses, please see our <a title='' href='http://concord.org/software-license' target=''>licenses page</a>. When sharing this resource, include attribution to the Concord Consortium and links to <a title='' href='http://concord.org/' target=''>concord.org</a> and the CC-BY-4.0 license. Copyright &copy; 2017 The Concord Consortium.</p><p class='footer-txt'>This Next-Generation Molecular Workbench activity was developed with a grant from <a href='http://www.google.org/'>Google.org</a>. The original <a href='http://mw.concord.org/modeler/'>Classic Molecular Workbench</a> was supported by a series of grants from the <a href='http://nsf.gov/'>National Science Foundation</a>.</p><p class='footer-txt'>Questions? Comments? You can reach us at <a href='mailto:nextgenmw@concord.org'>nextgenmw@concord.org</a>.</p>",
+    footerLogo: [],
   },
   {
     id: 2,
     name: "High Adventure Science",
-    logo: hasLogo,
+    projectLogo: hasLogo,
     url: "http://has.concord.org/",
+    footer: 
+    "<p>This <a title=\"High-Adventure Science | The Concord Consortium\" href=\"http://has.concord.org\">High-Adventure Science</a> activity was developed under a series of grants from the <a title=\"nsf.gov - National Science Foundation - US National Science Foundation (NSF)\" href=\"http://nsf.gov/\">National Science Foundation</a> (DRL-0929774, DRL-1220756 ) in partnership with the <a title=\"University of California, Santa Cruz\" href=\"http://www.ucsc.edu/\">University of California, Santa Cruz</a> and <a title=\"Teachers Homepage - National Geographic Education\" href=\"http://education.nationalgeographic.com/\">National Geographic Society</a>.</p><p>Copyright 2018 <a title=\"The Concord Consortium | Revolutionary digital learning for science, math and engineering\" href=\"http://concord.org/\">The Concord Consortium</a>. All rights reserved. This is licensed under a Creative Commons Attribution-NonCommercial-NoDerivs 3.0 Unported License. Please provide attribution to The Concord Consortium and the URL http://concord.org.</p><p>Questions? Comments? You can reach us at <a href=\"mailto:HAS@concord.org\">HAS@concord.org</a>.</p>",
+    footerLogo: [ concordLogo, natgeoLogo, ucscLogo, nsfLogo ],
   },
   {
     id: 3,
     name: "RITES",
-    logo: ritesLogo,
+    projectLogo: ritesLogo,
     url: "http://investigate.ritesproject.net/",
+    footer: "",
+    footerLogo:[],
```

Comment:
Super minor complaint, but we should add a space after the colon for consistency: `footerLogo: [],`.